code/modules/uplink/uplink_item.dm

```diff
@@ -1130,13 +1133,13 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	name = "Codespeak Manual"
 	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which sounds like random concepts and drinks to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. One use."
 	item = /obj/item/codespeak_manual
-	cost = 2
+	cost = 1 //lowered its outclassed by comms
```

Comment:
Not when your server rules don't force Security to play dumb.

I'm not sure where we are on that front, quite honestly, but I'd rather this be kept as it is.